ember/app/ui/cases/detail/work-items/edit/form/template.hbs.orig

```diff
@@ -0,0 +1,38 @@
+<h3>{{this.workItem.name}}</h3>
```

Comment:
this file should probably not be checked in

Reply:
same goes for yarn.lock.orig. Maybe we should add `*.orig` to `.gitignore`?

Reply:
Removed orig files and added to gitignore